axelrod/strategies/calculator.py

```diff
@@ -2,14 +2,14 @@
 from axelrod.action import Action
 from axelrod.player import Player
 
-from .axelrod_first import Joss
```

Comment:
Since Joss is a somewhat more well-known strategy than the others in the first tournament, maybe we should use `from .axelrod_first import FirstByJoss as Joss` instead?

Reply:
Sure good call!